ocs_ci/utility/rosa.py

```diff
@@ -58,17 +58,24 @@ def create_cluster(cluster_name, version, region):
     multi_az = "--multi-az " if config.ENV_DATA.get("multi_availability_zones") else ""
     cluster_type = config.ENV_DATA.get("cluster_type", "")
     provider_name = config.ENV_DATA.get("provider_name", "")
+    mode = config.ENV_DATA.get("mode", "")
```

Comment:
I recommend to call the option rosa_mode or something similar so that it is clear what the mode is used for.